src/js/devtools/App.jsx

```diff
@@ -25,12 +26,21 @@ const App = ({ tabId }) => {
   const { themeName: chromeTheme } = chrome.devtools.panels;
   const theme = chromeTheme === 'default' ? defaultTheme : darkTheme;
 
-  return (
-    <ThemeProvider theme={theme}>
-      <SidePanel tabId={tabId} />
-      <MainPanel tabId={tabId} />
-    </ThemeProvider>
-  );
+  const detectKnotxFragments = useSelector(({ pageData }) => (
+    pageData[tabId] && pageData[tabId].fragments
+      ? pageData[tabId].fragments
+      : false));
```

Comment:
I believe it can be replaced with:
```
const detectKnotxFragments = useSelector(({ pageData }) => (pageData[tabId] && pageData[tabId].fragments) || false)
```
WDYT?